internal/app/common/continuous_tester.go

```diff
@@ -0,0 +1,144 @@
+package common
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/PagerDuty/go-pagerduty"
+
+	"github.com/golang/glog"
+	"github.com/livepeer/stream-tester/messenger"
+)
+
+type (
+	IContinuousTester interface {
+		// Start test. Blocks until error.
+		Start(start func(ctx context.Context) (int, error), testDuration, pauseBetweenTests time.Duration) error
```

Comment:
What's the thinking behind having an error code and an error object returned here?

Reply:
To be honest I don't know, I think it was copy paster from recordtester into the vod tester. I just refactored something we already had in VOD tester 🙃 

Saying that, when I looked again in the code, I think we return always `0` or `1` for the error code, so I think it's safe to remove. Then, removed.